domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/finishlesson/domain/Types.kt

```diff
@@ -0,0 +1,19 @@
+package com.krzykrucz.elesson.currentlesson.finishlesson.domain
+
+import com.krzykrucz.elesson.currentlesson.topic.domain.LessonOrdinalNumber
+import com.krzykrucz.elesson.currentlesson.topic.domain.LessonTopic
+import java.time.LocalTime
+
+enum class LessonBell {
+    RANG, NOT_RANG
+}
+
+data class FinishedLesson private constructor(val lessonTopic: LessonTopic, val lessonOrdinalNumber: LessonOrdinalNumber) {
```

Comment:
I have a question about file names, what is our convention? Because now we have `FinishLessonTypes` instead of just `Types`. I do not hold any strong opinion which version is better but I think we should unify it.

Reply:
it was an initial approach, then I realised that the current convetion is toward `Types.kt` name so I renamed it accordingly.

Reply:
we can sort it out later

Reply:
:+1: